app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -28,12 +42,33 @@ fun PlotSurface() {
         modifier = Modifier.fillMaxSize(),
         color = Color.White
     ) {
-        // TODO Build out the plot surface
-        //  This should include a Column composable that
-        //  includes a Map, and two MapSlider composables
-        //  (one slider for each axis).
-    }
+        Column (modifier = Modifier.fillMaxSize(),
+                horizontalAlignment = Alignment.CenterHorizontally)
+            {
+            var xPercentage: Float by remember { mutableFloatStateOf(0.5f) }
+            var yPercentage: Float by remember { mutableFloatStateOf(0.5f) }
+            val boxSize = 300
+            val innerBoxSize = 20
+            val xOffset = 10
+            val yOffset = 10
```

Comment:
Love how all the dimensions are declared, without any hardcoding further down in the file.